### PR TITLE
Updated some 1.10.0 things

### PR DESCRIPTION
**Pull request by StrangeOne101**
- Converted to PK's TempBlock class
- Removed cooldown enforcer (as cooldowns are enforced by PK now)
- Updated FrostBreath to work in custom biomes

I'd also appreciate if the PR or repo could be used for hacktoberfest ❤️

### CODE REVIEW

**Comment by StrangeOne101** [comment]
@Aztlon Anything I need to touch on?

File: src/com/jedk1/jedcore/JCManager.java [hunk 1]
```diff
@@ -6,9 +6,8 @@
  import com.jedk1.jedcore.ability.waterbending.HealingWaters;
  import com.jedk1.jedcore.ability.waterbending.IcePassive;
  import com.jedk1.jedcore.util.RegenTempBlock;
- import com.jedk1.jedcore.util.TempFallingBlock;
 
-public class JCManager implements Runnable {
+ public class JCManager implements Runnable {
```
Comment (Aztlon) [inline src/com/jedk1/jedcore/JCManager.java:10]: AHHHHHHHHH

File: pom.xml [hunk 2]
```diff
@@ -17,6 +17,7 @@
         <id>jitpack.io</id>
         <url>https://jitpack.io</url>
       </repository>
+
```
Comment (Aztlon) [inline pom.xml:20]: why

**Review by Aztlon — APPROVED** [review]

File: src/com/jedk1/jedcore/JCManager.java (hunk 1 above)
Comment (StrangeOne101) [reply src/com/jedk1/jedcore/JCManager.java:10]: I DONT KNOW HOW THIS HAPPENED, ILL FIX IT

File: pom.xml (hunk 2 above)
Comment (StrangeOne101) [reply pom.xml:20]: My IDE hates you, that's why. I'll delete

**Pull request merged**